code/style.css

```diff
@@ -1,4 +1,288 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  background:white;
+  color: rgb(78, 77, 77);
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 15px;
+  font-weight: normal;
 }
+
+.navigator {
+  background-color: whitesmoke;
+  height: 125px;
+  padding: 15px 0px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  position: relative;
+  align-items: center;
+}
+
+.fa-sun {
+  color: rgb(78, 77, 77);
+  font-size: 45px;
+  position: absolute;
+	left: 20px;
+}
+
+h1 {
+  font-size: 38px;
+  position: absolute;
+	left: 100px;
+}
+
+.nav-text1, .nav-text2 {
+  display: none;
+  position: absolute;
+  margin: 0px;
+  font-size: 18px;
+}
+
+.nav-text1 {
+  right: 140px;
+}
+
+.nav-text2 {
+  right: 50px;
+}
+
+.fa-bars {
+  position: absolute;
+  right: 60px;
+  display: none;
+  font-size: 25px;
+}
+
+.bars {
+  display: none;
+}
+
+.big-news {
+  margin: 10px 10px 0 10px;
+  display: flex;
+  flex-direction: column;
+  background-color: whitesmoke;
+  position: relative;
+  align-items: center;
+}
+
+.gotland-image {
+  width: 100%;
+  height: auto;
+}
+
+.big-news-text:hover {
+  background-color:white;
+  text-shadow: 1px 1px grey;
+}
+
+h2 {
+  font-size: 28px;
+  text-transform: uppercase;
+  margin: 10px;
+}
+
+.small-text-big-news {
+  margin-top: 0px;
+  font-size: 20px;
+  margin: 0px 10px;
+}
+
+h3 {
+  font-size: 24px;
+  text-transform: uppercase;
+  margin: 10px 5px 5px 5px;
+}
+
+p {
+  margin: 0px 5px 25px 5px;
+}
+
+p:hover {
+  text-shadow: 1px 1px grey;
+}
+
+.more-news {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.news-card {
+  background-color: whitesmoke;
+  width: 100%;
+  padding-top: 5px;
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  width: 80%;
+}
+
+.news-card:hover {
+  background-color: white;
+  border-top: 2px solid coral;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: whitesmoke;
+  padding: 30px;
+  animation-name: disco-footer;
```

Comment:
Line 140-142 is very clear and efficiently done. I will definitely check out next time I want to make the disco footer. Thanks!